src/components/description-list/DescriptionList.vue

```diff
@@ -1,11 +1,12 @@
 <template>
     <VsRow
-        tag="dl"
         class="vs-description-list"
         :class="{ 'list-inline': inline }"
     >
-        <!-- @slot The description content goes here -->
-        <slot />
+        <dl>
```

Comment:
Looks like this might need an inline or something, there's a bit of a layout change

Reply:
Looks like adding the dl wrapper messed up the flexbox - should be fixed now